strongbox-data-service/src/main/resources/orientdb_MEMORY.properties

```diff
@@ -0,0 +1,4 @@
+strongbox.orientdb.protocol = memory
+strongbox.orientdb.database = strongbox
+strongbox.orientdb.username = admin
+strongbox.orientdb.password = password
```

Comment:
Please, make sure there are empty lines at the end of such resource files. (Here and below).  :)

Reply:
done

Reply:
Thanks! :)